pkg/manager/member/pd_member_manager.go

```diff
@@ -812,10 +811,10 @@ func getNewPDSetForTidbCluster(tc *v1alpha1.TidbCluster, cm *corev1.ConfigMap) (
 
 func getPDConfigMap(tc *v1alpha1.TidbCluster) (*corev1.ConfigMap, error) {
 	// For backward compatibility, only sync tidb configmap when .tidb.config is non-nil
-	config := tc.Spec.PD.Config
-	if config == nil {
+	if tc.Spec.PD.Config == nil {
```

Comment:
What about TiCDC and TiFlash?

Reply:
seem to not need to config `[security]` when tls is enable

Reply:
ticdc use cmd args without config, and tiflash have been used `DeepCopy()`

Reply:
TiCDC also supports `config` from v1.2.x.

Reply:
Done